### PR TITLE
Feature achieved work section mobile

### PR DESCRIPTION
**Pull request by aimemalaika**
## What does this PR do?
This PR is raised to to create a section that list project worked on
## Description of the task to be completed
- list projects
- Display project detail
## How should it be tested
- clone the repository\
  `git clone https://github.com/aimemalaika/portofolio.git`
- navigate to the folder\
  `cd portfolio`
- checkout the feature\
  `feature-achieved-work-section-mobile`

### CODE REVIEW

**Review by enaburekhan — COMMENTED** [review]
Hi Aimemalaika, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project with no descriptive Pull request. Pull requests should have a short summary describing introduced changes in more detail. check this [page](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) on pathwright on how to write a descriptive PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by aimemalaika** [comment]
Hello Eric,
do you mean this description? if yes please let me know so that i can request another review. thanks.\
Screen Shot 2021-11-25 at 15 05 11

**Comment by enaburekhan** [comment]
Hello Aimemalaika,
Yes something like this:
image
 But in this case, the project is already completed. Also, check the link I shared for more hints.

File: index.html [hunk 1]
```diff
@@ -56,6 +57,136 @@ <h1 class="describe">
       </button>
     </div>
   </section>
+  <section class="achieved-work">
+    <h1 class="section-title">My Recent Work</h1>
```
Comment (VanessaAoki) [inline index.html]: - [x] To keep the HTML semantic, you should consider changing this to a `<h2>` tag instead. Even though using more than one `<h1>` is allowed by the HTML specification, it is not considered a best practice, you can read more about headings [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).

File: index.html [hunk 2]
```diff
@@ -56,6 +57,136 @@ <h1 class="describe">
...
+          the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+          scrambled it 1960s with the relea
+        </p>
+        <ul class="modal-buttons">
+          <li><a href="#" class="success-button medium-size">See Live <i class="fa fa-external-link"></i></a></li>
+          <li><a href="#" class="success-button medium-size">See Source <i class="fa fa-github"></i></a></li>
+        </ul>
+      </div>
+    </div>
+  </section>
```
Comment (VanessaAoki) [inline index.html:179]: - [ ] Great job implementing the whole "works" section. According to the requirements, you should also do the "about me" section, as you can see here:

image

File: assets/css/style.css [hunk 3]
```diff
@@ -172,4 +173,227 @@ body {
...
+  font-size: 24px;
+  line-height: 32px;
+  margin: 16px auto;
+}
+
+.achievementlist {
+  list-style-type: none;
+  display: grid;
+  padding: 0;
+}
```
Comment (VanessaAoki) [inline assets/css/style.css:252]: - [ ] Great job using `display: grid` as required, but right now, your project is displaying one work per row. In order to match the original design better, you should consider styling them to be displayed 3 per row, with at least 2 rows.

Figma design:
image


Your design:
image

**Review by VanessaAoki — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @aimemalaika, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 

## Required Changes ♻️
_Check the comments under the review._


You can also consider:
- [x] Adding the resolution used to style your project in the PR comments, so code reviewers can see whatever you see as well.

so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/css/style.css (hunk 3 above)
Comment (aimemalaika) [inline assets/css/style.css]: But we are only working on the mobile version of the web

Comment (VanessaAoki) [reply assets/css/style.css]: You are right, I'm sorry, I'm deleting this comment. But you should work on the other comments!

**Review by enaburekhan — APPROVED** [review]
Hi Aimemalaika, 

Thank you for making the changes requested by the last reviewer. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**